provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -329,9 +332,18 @@ def execute_sql_query(job_type, output_uri, sql_query, spark_configs, source_lis
         # remove the '/' at the end of output_uri in order to avoid double slashes in the output file path.
         output_uri_with_timestamp = f"{output_uri.rstrip('/')}/{safe_datetime}"
 
-        output_dataframe.write.option("header", "true").mode("overwrite").parquet(
-            output_uri_with_timestamp
-        )
+        if output_format == "parquet":
+            output_dataframe.write.option("header", "true").mode("overwrite").parquet(
+                output_uri_with_timestamp
+            )
+        elif output_format == "csv":
```

Comment:
I am assuming that we don't include the headers because of the s3 import. I wonder if it makes sense to have another flag for it because in the future we might want to write as csv. Or I guess once you add your comment, it should make sense.

Reply:
I actually think I might be able to add the headers back in after changing the `SELECT` statement in the materialization queries. Let me give that a shot so this flag is responsible for one thing, which would of course be better. Stay tuned.

Reply:
OK, I spoke too soon: we're using slightly different naming conventions for keys in DynamoDB, so I will need to provide a list of headers at the time of import. I'll look into adding a `--headers` flag to the runner script.

Reply:
`--headers include/exclude` arg has been added to direct the runner script whether to write to the output file. I've added an exception if used for parquet purely given our iterators rely on the headers.